tests/VM_with_planetary_network/planetary_network_test.go

```diff
@@ -47,17 +46,11 @@ func TestSingleNodeDeployment(t *testing.T) {
 	yggIP := terraform.Output(t, terraformOptions, "ygg_ip")
 	assert.NotEmpty(t, yggIP)
```

Comment:
is there some sort of validation on the value of yggIP?

Reply:
we can check if there is library like regex for this case to know the output format
but i think if the value is false will lead the ssh to machine will fail